tests/testthat/test_time.R

```diff
@@ -0,0 +1,135 @@
+library(testthat)
+library(recipes)
+library(lubridate)
+library(tibble)
+
+
+examples <- data.frame(
+  times = ymd_hms("2022-05-06 10:01:07") +
+  hours(1:5) + minutes(1:5) + seconds(1:5)
+)
+
+date_rec <- recipe(~ times, examples) %>%
+  step_time(all_predictors())
+
+feats <- c("ampm", "hour24", "hour12", "minute", "second", "decimal_day")
```

Comment:
I think we should enforce Jenny's testing suggestions in all new test files. So

- We can `library(testthat)` for Max, I guess
- No need to `library(recipes)` because even Max will do that
- Definitely avoid `library(lubridate)` and `library(tibble)` as that can have far reaching side effects

I see nothing wrong with duplicating these data objects in each of the tests. If you feel strongly against that, I'd create `helper-time.R` and add something like `time_example()` that makes the data frame, `time_recipe()` that makes the recipe and `time_features()` that make the features vector.

Reply:
Thank you, done!